spec/post_spec.rb

```diff
@@ -0,0 +1,58 @@
+require_relative './rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) do
+    User.create(
+      name: 'Loren',
+      photo: 'https://this-person-does-not-exist.com/en/download-page?image=genef4e9868ae582ca3061881b69d8fbeb1',
+      posts_counter: 0,
+      bio: 'I am Loren, and I love sport.'
+    )
+  end
```

Comment:
- [ ] You did a great job working on the changes the previous reviewer suggested! 💯 However, not every suggested change is implemented. Please make sure to test your custom methods inside your `Post` model, such as `recent_five_comments`. 🤓 🔧